### PR TITLE
Add meshes to model

### PR DESCRIPTION
**Pull request by LorenzoFiori**
As per the title. I updated the model with meshes for the body and muscle linkages. Obviously, these scale with the model's size.

### CODE REVIEW

File: code/src/modifyMeshModels.py [hunk 1]
```diff
@@ -0,0 +1,98 @@
...
+        if link_name in scalingParamMesh:
+            scale = scalingParamMesh[link_name]
+            original_visual = link.visuals[0]
+            original_origin = original_visual.origin
+
+            visual = Visual(
+                name=link_name,
+                geometry=Geometry(
+                    mesh=Mesh(
+                        filename=f"{mesh_folder}\{link_name}.stl",
```
Comment (claudia-lat) [inline code/src/modifyMeshModels.py]: With \ I got the following error:

../human-model-generator/code/src/modifyMeshModels.py:65: SyntaxWarning: invalid escape sequence '\{'
  filename=f"{mesh_folder}\{link_name}.stl",
../human-model-generator/code/src/modifyMeshModels.py:84: SyntaxWarning: invalid escape sequence '\{'
  filename=f"{mesh_folder}\{muscle}.stl",

File: code/src/modifyMeshModels.py [hunk 2]
```diff
@@ -0,0 +1,98 @@
...
+
+        if link_name in map_link_to_muscles:
+            muscles = map_link_to_muscles[link_name]
+            for muscle in muscles:
+                muscle_scale = scalingParamMesh[link_name]
+                muscle_visual = Visual(
+                    name=muscle,
+                    geometry=Geometry(
+                        mesh=Mesh(
+                            filename=f"{mesh_folder}\{muscle}.stl",
```
Comment (claudia-lat) [inline code/src/modifyMeshModels.py]: With \ I got the following error:

../human-model-generator/code/src/modifyMeshModels.py:65: SyntaxWarning: invalid escape sequence '\{'
  filename=f"{mesh_folder}\{link_name}.stl",
../human-model-generator/code/src/modifyMeshModels.py:84: SyntaxWarning: invalid escape sequence '\{'
  filename=f"{mesh_folder}\{muscle}.stl",

**Review by claudia-lat — CHANGES_REQUESTED** [review]

File: code/src/modifyMeshModels.py (hunk 1 above)
Comment (LorenzoFiori) [reply code/src/modifyMeshModels.py]: Fixed

File: code/src/modifyMeshModels.py (hunk 2 above)
Comment (LorenzoFiori) [reply code/src/modifyMeshModels.py]: Fixed

**Comment by claudia-lat** [comment]
@LorenzoFiori  after the last commit https://github.com/ami-iit/human-model-generator/pull/20/commits/e18c842e5909f28976e7194acd866f7ea2f35f05, I launched the code with the following options:
````
OPT_CHECK_CONSISTENCY_MODEL = False
OPT_VISUALIZZATION_MODEL = True
````
and I got this error
````
File ".../human-model-generator/code/main.py", line 250, in <module>
    viz.modelViz("ModelVisualizer").setPositions(G_T_base, s)
                                                 ^^^^^^^^
NameError: name 'G_T_base' is not defined
````

**Review by claudia-lat — APPROVED** [review]

**Pull request merged**